<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmPrincipal</class>
 <widget class="QMainWindow" name="FrmPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtOpenGL v1.00</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbFps">
        <property name="text">
         <string>FPS: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbSize">
        <property name="text">
         <string>Size [width: 0, height: 0]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="WidgetGL" name="widgetGL">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WidgetGL</class>
   <extends>QOpenGLWidget</extends>
   <header>WidgetGL.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
